pyodide_build/cli/config.py

```diff
@@ -19,6 +21,13 @@
     "meson_cross_file": "MESON_CROSS_FILE",
 }
 
+# Load the values of sysconfig.get_paths() with a
+# "pyodide_sysconfig_" prefix to differentiate them
+# from other configuration variables
+PYODIDE_CONFIGS.update(
+    {f"pyodide_sysconfig_{k}": v for k, v in sysconfig.get_paths().items()}
```

Comment:
There are only eight values in `sysconfig.get_paths()`, so, I think it is okay to list all of them, which will be more explicit and make the code more readable.